src/main/java/org/isf/accounting/gui/PatientBillEdit.java

```diff
@@ -1971,8 +1917,78 @@ public void setValueAt(Object value, int rowIndex, int columnIndex) {
 		}
 	}
 
+	private final class JTableTotalModel extends DefaultTableModel {
+
+		private static final long serialVersionUID = 1L;
+		Class< ? >[] types = new Class< ? >[] { JLabel.class, JLabel.class, Double.class, };
+
+		private JTableTotalModel() {
+			super(new Object[][] { { "<html><b>" + MessageBundle.getMessage("angal.common.total.txt").toUpperCase() + "</b></html>", currencyCod, total } },
+							new String[] { "", "", "" });
+		}
+
+		private JTableTotalModel(Object[][] data, Object[] columnNames) {
+			super(data, columnNames);
+		}
+		@Override
+		public Class< ? > getColumnClass(int columnIndex) {
+			return types[columnIndex];
+		}
+		@Override
+		public boolean isCellEditable(int row, int column) {
+			return false;
+		}
+	}
+
 	public String formatDateTime(LocalDateTime time) {
 		return DATE_TIME_FORMATTER.format(time);
 	}
 
+	private final class JTableBalanceModel extends DefaultTableModel {
+
+		private static final long serialVersionUID = 1L;
+		Class< ? >[] types = new Class< ? >[] { JLabel.class, JLabel.class, Double.class, };
+
+		private JTableBalanceModel() {
+			super(new Object[][] {
+					{ "<html><b>" + MessageBundle.getMessage("angal.newbill.balance.txt").toUpperCase() + "</b></html>", currencyCod, balance } },
+							new String[] { "", "", "" });
+		}
+
+		private JTableBalanceModel(Object[][] data, Object[] columnNames) {
+			super(data, columnNames);
+		}
+
+		@Override
+		public Class< ? > getColumnClass(int columnIndex) {
+			return types[columnIndex];
+		}
+
+		@Override
+		public boolean isCellEditable(int row, int column) {
+			return false;
+		}
+	}
+
+	private final class JTableBigTotalModel extends DefaultTableModel {
+
+		private static final long serialVersionUID = 1L;
+		Class< ? >[] types = new Class< ? >[] { JLabel.class, JLabel.class, Double.class, };
+
+		private JTableBigTotalModel() {
+			super(new Object[][] { { "<html><b>" + MessageBundle.getMessage("angal.newbill.topay.txt") + "</b></html>", currencyCod, bigTotal } },
+							new String[] { "", "", "" });
+		}
+		private JTableBigTotalModel(Object[][] data, Object[] columnNames) {
```

Comment:
Why was this `private` method added as it is never called?

Reply:
ok, removing... also for `JTableBalanceModel` and `JTableTotalModel`